tests/sentry/grouping/snapshots/test_categorization/test_categorization/block_invoke.pysnap

```diff
@@ -15,9 +15,9 @@ _:_ (thread_id:_, crashed:_)
                               CoreFoundation  __CFRunLoopRun category=internals
                               CoreFoundation  __CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__ category=internals
                            libdispatch.dylib  _dispatch_main_queue_callback_4CF category=internals
-                           libdispatch.dylib  _dispatch_client_callout category=threadbase
```

Comment:
I have no idea what this change does, but we're good with it? Is `threadbase` something which only applies with mobile?

Reply:
My comments got lost when I split things up into various PRs.

I removed a bad rule in the newstyle.txt that accounts for this change:
https://github.com/getsentry/sentry/pull/61675

Reply:
Got it. How have these tests been passing since then, though?

Reply:
Good question. I had made this a separate PR so it would be obvious.

If you go the test_categorization.py file you will see that I have changed this:
<img width="805" alt="image" src="https://github.com/user-attachments/assets/f366d234-312c-42cf-988f-330407bfed24">